vms/registration/passwordmatch.py

```diff
@@ -0,0 +1,6 @@
+# account can only be created if both passwords match
```

Comment:
something like this should not go into a place like this. maybe to registration/utils/password_match.py (or even better - just put it in the form validation file) ?